.gitignore

```
@@ -0,0 +1 @@
+!.git/workflows/*
```

Shouldn't this be the `github` folder, instead?
```suggestion
!.github/workflows/*
```